### PR TITLE
operator-mathOperator

### PR DESCRIPTION
**Pull request by boajer**

### CODE REVIEW

File: views/js/scoring/processor/expressions/operators/mathOperator.js [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+        sin: Math.sin,
+        cos: Math.cos,
+        tan: Math.tan,
+        sec: reciprocalFactory(Math.cos),
+        csc: reciprocalFactory(Math.sin),
+        cot: reciprocalFactory(Math.tan),
+        asin: Math.asin,
+        acos: Math.acos,
+        atan: Math.atan,
+        atan2: {
```
Comment (krampstudio) [inline views/js/scoring/processor/expressions/operators/mathOperator.js]: Why do you use the property `func` inside ?

Comment (boajer) [inline views/js/scoring/processor/expressions/operators/mathOperator.js]: I realized that is not very transparent naming was used there, it is for splitting non standard cases, when function require more than one argument ( like `atan2` ), originally  there are was extra property with number of required arguments

Comment (krampstudio) [inline views/js/scoring/processor/expressions/operators/mathOperator.js]: ok I'll just rename it and add a comment

**Comment by krampstudio** [comment]
I've also a test case that fails : 

```
>> Process - mathOperator [exp]
>> Message: The mathOperator value is correct with accuracy 1e-15
>> Actual: 15.154262241479262
>> Expected: 15.154262241479266
```

**Pull request merged**